### ISSUE TITLE
Can we add border radius to wave?

### ISSUE THREAD

**Issue by 91priyansh** [RutvikRana/music_slider#1]
i would to like to add border radius to wave. is it possible to do?

**Comment by RutvikRana** [comment]
Yes, we can. Music bars are simple containers, so use Borderradius with
Boxdecoration. Add it to script. I can’t do it now because of my
examination. I will add this feature in update.

**Comment by 91priyansh** [comment]
@RutvikRana thanks will do it
